### PR TITLE
Simplify FundsLocker a bit

### PR DESCRIPTION
**Pull request by Krigpl**
`TaskFundsLock` doesn't need to know about `transaction_system`.
Also fixed corresponding test.

### CODE REVIEW

File: tests/golem/transactions/ethereum/test_fundslocker.py [hunk 1]
```diff
@@ -13,9 +12,9 @@ class TestFundsLocker(TempDirFixture):
     def setUp(self):
         super().setUp()
         self.ts = mock.MagicMock()
-        self.ts.eth_for_batch_payment.return_value = 13000
+        self.ts.eth_for_batch_payment.side_effect = lambda n: n * 13000
```
Comment (maaktweluit) [inline tests/golem/transactions/ethereum/test_fundslocker.py:15]: These values could be stored in a shared variable for easy updates later on.

They could even link to the constants in SCI, any reason you did not?

**Review by maaktweluit — APPROVED** [review]
LGTM!

One small q, not blocking...

File: tests/golem/transactions/ethereum/test_fundslocker.py (hunk 1 above)
Comment (Krigpl) [reply tests/golem/transactions/ethereum/test_fundslocker.py:15]: Yeah, that wasn't me who introduced this code :)

Comment (Krigpl) [reply tests/golem/transactions/ethereum/test_fundslocker.py:15]: But yeah, these could be non magic numbers - someday...

**Pull request merged**